src/NuGet/BuildNuGets.csx

```diff
@@ -122,6 +124,8 @@ string[] TestPackageNames = {
 var PreReleaseOnlyPackages = new HashSet<string>
 {
     "Microsoft.CodeAnalysis.Build.Tasks",
+    "Microsoft.CodeAnalysis.CSharp.CodeStyle",
+    "Microsoft.CodeAnalysis.VisualBasic.CodeStyle",
```

Comment:
Yes 

CodeFlow.

---
In reply to: [109054195](https://github.com/dotnet/roslyn/pull/18285#discussion_r109054195) [](ancestors = 109054195)